templates/common/.devcontainer/Dockerfile/postgresql/Dockerfile

```diff
@@ -0,0 +1,7 @@
+ARG VARIANT=bullseye
+FROM --platform=amd64 mcr.microsoft.com/vscode/devcontainers/base:0-${VARIANT}
+RUN export DEBIAN_FRONTEND=noninteractive \
+     && apt-get update && apt-get install -y xdg-utils \
+     && apt-get update && apt-get install --yes --no-install-recommends postgresql-client \
```

Comment:
Do we not have a ghcr feature for this?

Reply:
Didn't find postgresql feature in https://github.com/devcontainers/features/tree/main/src. So do it in Dockerfile.

Reply:
> Do we not have a ghcr feature for this?

Didn't find postgresql feature in https://github.com/devcontainers/features/tree/main/src. So do it in Dockerfile.